sources/models/src/aws-dev/defaults.d/50-aws-dev.toml

```diff
@@ -1,12 +1,3 @@
-[configuration-files.containerd-config-toml]
-# No override to path
-template-path = "/usr/share/templates/containerd-config-toml_basic"
-
-# Docker
-[services.docker]
-restart-commands = ["/bin/systemctl try-restart docker.service"]
-configuration-files = ["proxy-env"]
-
 # Metrics
 [settings.metrics]
```

Comment:
It's weird that these metrics settings are in `50-aws-dev.toml`, overriding `30-metrics.toml` in defaults.d.

`vmware-dev` also includes `30-metrics.toml` but - apparently incorrectly? - does not override any of the values, so it diverges from `aws-dev` behavior.

I'd also argue that using shared defaults for `kubernetes-network` and `kubernetes-metrics` are a mistake and that these are inherently variant-specific since they have to know the full set of expected services.

The `vmware-k8s` variants should care about the status of the `vmtoolsd` service, but won't be able to express that without essentially replacing and duplicating `kubernetes-metrics`. I don't think we have an equivalent for `kubernetes-network` today, but it's not hard to imagine a platform-specific service that needs proxy settings.

I'd suggest merging both `kubernetes-network` and `kubernetes-metrics` into `kubernetes-aws`, and living with slight redundancy when we add `kubernetes-vmware`.